File: .sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// Thesmsworks SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class ThesmsworksMakeContext
{
    public static function call(array $ctxmap, ?ThesmsworksContext $basectx): ThesmsworksContext
    {
        return new ThesmsworksContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/csharp/utility/MakeContext.cs
// Thesmsworks SDK utility: makeContext.

namespace ThesmsworksSdk.Util;

public static partial class SdkUtility
{
    internal static Context MakeContextUtil(Dictionary<string, object?>? ctxmap, Context? basectx)
    {
        return new Context(ctxmap, basectx);
    }
}
